unitary/examples/quantum_rpg/main_loop.py

```diff
@@ -92,6 +97,14 @@ def loop(self, user_input: Optional[Sequence[str]] = None) -> None:
             if cmd is not None:
                 self.world.move(cmd)
                 continue
+            words = current_input.split()
+            action = self.world.current_location.get_action(words[0])
```

Comment:
This appears to throw on an empty line

Reply:
Removed.

Reply:
There is still the same problem; the code that throws on an empty line is just in another function.

LGTM other than that

Reply:
Fixed, with test.  I misunderstood what you meant at first.